docs/developing/config.md

```diff
@@ -2,158 +2,57 @@
 title: Config
```

Comment:
Could we also backport the change to the v6 docs?

Reply:
Done in https://github.com/ionic-team/ionic-docs/pull/2727/commits/5ca3b5075b6de157f6e1b49c72c1249b8d069f28